packages/vm/vmimpl/send.go

```diff
@@ -14,12 +14,18 @@ func (vmctx *vmContext) getNFTData(chainState kv.KVStore, nftID iotago.ObjectID)
 }
 
 func (reqctx *requestContext) send(params isc.RequestParameters) {
+	// TODO: Here used to be Cross-Chain logic.
+	// Look at ac543caf8ddb276d5580c3ef5d4a41c6ef90e625 to see how it was implemented in the past.
+	// For now removed everything related to it.
```

Comment:
I don't think this comment will ever be needed. At least I would remove the TODO, but really I would remove it all.